private/model/api/api.go

```diff
@@ -694,6 +708,10 @@ func (a *API) ServiceGoCode() string {
 	}
 	a.AddSDKImport("private/protocol", a.ProtocolPackage())
 
+	if a.HasServiceClientConfig() {
+		a.AddSDKImport("service", a.PackageName(), a.ConfigPackageName())
+	}
+
```

Comment:
Nit: Conditional AddSDKImport probably can be a part of the template.

Reply:
This is included in the most recent commit.

Reply:
It could be, but I don't agree with adding unnecessarily adding complexity to the cognitive load of understanding the template for a single import.